frontend/src/pages/AlertDetails/AlertHeader/ActionButtons/index.tsx

```diff
@@ -0,0 +1,64 @@
+import './actionButtons.styles.scss';
+
+import { Button, Divider, Dropdown, MenuProps, Tooltip } from 'antd';
+import { Copy, Ellipsis, PenLine, Trash2 } from 'lucide-react';
+import CopyToClipboard from 'periscope/components/CopyToClipboard';
+import React from 'react';
+
+const menu: MenuProps['items'] = [
+	{
+		key: 'rename-rule',
+		label: 'Rename',
+		icon: <PenLine size={16} color="var(--bg-vanilla-400" />,
+		onClick: (): void => {},
+	},
+	{
+		key: 'duplicate-rule',
+		label: 'Duplicate',
+		icon: <Copy size={16} color="var(--bg-vanilla-400" />,
+		onClick: (): void => {},
+	},
+];
+
+const menuStyle: React.CSSProperties = {
+	padding: 0,
+	boxShadow: 'none',
+	fontSize: 14,
+};
+
+function AlertActionButtons(): JSX.Element {
+	return (
+		<div className="alert-action-buttons">
+			<CopyToClipboard textToCopy={window.location.href} />
+
+			<Divider type="vertical" />
+
+			<Dropdown
+				trigger={['click']}
+				menu={{ items: menu }}
+				// eslint-disable-next-line react/no-unstable-nested-components
+				dropdownRender={(menu): JSX.Element => (
+					<div className="dropdown-menu">
```

Comment:
defining components in the props directly itself has performance downsides. More details here - https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unstable-nested-components.md